.github/workflows/create-portable-zip.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import os
+import sys
+import zipfile
+
+import gha
+
+if len(sys.argv) != 3:
+    gha.print_error('Usage: create-portable-zip.py <output_path> <release|debug>')
+    sys.exit(1)
+else:
+    output_path = sys.argv[1]
+    configuration = sys.argv[2].lower()
+
+with zipfile.ZipFile(output_path, 'x', zipfile.ZIP_DEFLATED, compresslevel=9) as output:
+    output.mkdir('Extensions')
+    output.mkdir('Gallery')
+
+    output.write(f'artifacts/bin/Bonsai/{configuration}-repacked/Bonsai.exe', 'Bonsai.exe')
+    output.write(f'artifacts/bin/Bonsai/{configuration}/Bonsai32.exe', 'Bonsai32.exe')
+
+    nuget_config = [
+        '<?xml version="1.0" encoding="utf-8"?>',
+        '<configuration>',
+        '  <packageSources>',
+        '    <add key="Gallery" value="Gallery" />',
+        '    <add key="Bonsai Packages" value="https://www.myget.org/F/bonsai/api/v3/index.json" />',
+        '    <add key="Community Packages" value="https://www.myget.org/F/bonsai-community/api/v3/index.json" />',
+    ]
+
+    nuget_api_url = os.getenv('NUGET_API_URL')
+    if nuget_api_url is not None:
+        nuget_config.append(f'    <add key="NuGet Package Testing Feed" value="{nuget_api_url}" />')
+
+    # Unstable builds of Bonsai will automatically reference the GitHub Packages feed
+    if os.getenv('IS_FULL_RELEASE') == 'false':
```

Comment:
As mentioned during our meeting, it's not really that much (it just looks like a lot since I included instructions to work around https://github.com/bonsai-rx/bonsai/issues/1849)

More important that the GitHub Packages aspect though is that "NuGet Package Testing Feed". That feature is invaluable for testing this workflow in forks and I'd like to keep it.